projects/aca-content/src/lib/services/node-actions.service.ts

```diff
@@ -563,6 +563,47 @@ export class NodeActionsService {
     return null;
   }
 
+  getInfoDrawerIcon(node: Node) {
```

Comment:
Please add a return type for all methods.

Reply:
Added